std/conv.d

```diff
@@ -5983,3 +5983,14 @@ if ((radix == 2 || radix == 8 || radix == 10 || radix == 16) &&
         }
     }
 }
+
+// Converts an unsigned integer to a compile-time string constant.
+package enum toCtString(ulong n) = n.stringof[0 .. $ - "LU".length];
```

Comment:
Shouldn't this be `package(std)` or perhaps discuss about making this a public interface?

Reply:
`package(std)` means the same thing as `package` in this context.

Since the original symbol is private, making it public is a functional change, not a refactor.

Reply:
Ah, right, makes sense